.circleci/config.yml

```diff
@@ -193,40 +191,38 @@ jobs:
             fi
       - run:
           name: Run FitLins
-          no_output_timeout: 2h
+          no_output_timeout: 4h
           command: |
             mkdir -p /tmp/ds003/work /tmp/ds003/derivatives
             chmod 777 /tmp/ds003/work /tmp/ds003/derivatives
             export CONDA_PREFIX=/opt/miniconda-latest/envs/neuro
-            docker run --rm -v /tmp/data/ds003_fmriprep:/data:ro \
+            docker run --rm -v /tmp/data/fitlins_tests:/data:ro \
                 -v /tmp/ds003/derivatives:/out \
                 -v /tmp/ds003/work:/scratch \
-                -v /tmp/data/ds003_models:/models \
-                -e COVERAGE_FILE=/scratch/.coverage \
+                -e COVERAGE_FILE=/scratch/.coverage_<< parameters.test_name >> \
                 --entrypoint="/neurodocker/startup.sh" \
                 poldracklab/fitlins:latest \
                 coverage run \
-                --rcfile /src/fitlins/docker/multiproc.coveragerc \
-                $CONDA_PREFIX/bin/fitlins /data/sourcedata /out dataset \
-                -d /data -m /models/model-001_smdl.json -w /scratch \
-                --participant-label 01 02 03 --space MNI152NLin2009cAsym \
-                --estimator << parameters.estimator >> \
-                --n-cpus 2 --mem-gb 4 -vv
+                --rcfile /src/fitlins/fitlins/tests/multiproc.coveragerc \
+                $CONDA_PREFIX/bin/pytest --cov=fitlins \
+                --cov-config /src/fitlins/fitlins/tests/multiproc.coveragerc \
+                --cov-report xml:/scratch/.coverage_pytest_<< parameters.test_name >>.xml \
```

Comment:
Seems a bit odd to `coverage run pytest --cov`. Should this just be `pytest`?

Reply:
Part of trying to figure out why the coverage from the new tests isn't being correctly captured. It didn't help so it can likely revert to just pytest. I'm hopeful that the most recent commit will have resolved the coverage issues.